build.gradle

```diff
@@ -394,10 +400,10 @@ dependencies {
   implementation 'net.minidev:json-smart:2.4.11'
   implementation group: 'com.auth0', name: 'java-jwt', version: '4.4.0'
 
-  testImplementation (group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.12') {
+  testImplementation (group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.22.2') {
     exclude group: 'com.github.hmcts.befta-fw', module: 'befta-fw'
   }
-  testImplementation group: 'com.github.hmcts', name: 'befta-fw', version: '8.8.5'
+  testImplementation group: 'com.github.hmcts', name: 'befta-fw', version: '9.0.2'
```

Comment:
dont think befta and test definitions is needed for testImplementation and we have it declared lower for functionalTestImplementation